.github/workflows/build-and-deploy.yaml

```
@@ -12,10 +12,13 @@ jobs:
         uses: actions/checkout@v4
         with:
           submodules: true
-      - name: Get the version
-        id: get_version
-        run: echo "::set-output name=version::$(tr -d '\n' < api/VERSION)"
-        shell: bash
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: aweber/imbi
+          tags: |
+            type=semver,pattern={{tag}}
```

@in-op I'm finally getting around to doing the next imbi release and noticed this PR. Should this be:

```suggestion
            type=semver,pattern={{raw}}
```

based on https://github.com/AWeber-Imbi/imbi-schema/commit/2ee38b14f9419372a0be683dd961735364b309dd